integration-tests/selenium/selenium.spec.js

```diff
@@ -0,0 +1,108 @@
+const { exec } = require('child_process')
+
+const { assert } = require('chai')
+const getPort = require('get-port')
+
+const {
+  createSandbox,
+  getCiVisAgentlessConfig
+} = require('../helpers')
+const { FakeCiVisIntake } = require('../ci-visibility-intake')
+const {
+  TEST_BROWSER_DRIVER,
+  TEST_BROWSER_NAME,
+  TEST_BROWSER_VERSION,
+  TEST_BROWSER_DRIVER_VERSION,
+  TEST_IS_RUM_ACTIVE,
+  TEST_TYPE
+} = require('../../packages/dd-trace/src/plugins/util/test')
+
+const webAppServer = require('../ci-visibility/web-app-server')
+
+const versionRange = ['4.11.0', 'latest']
+
+versionRange.forEach(version => {
+  describe(`selenium ${version}`, () => {
+    let receiver
+    let childProcess
+    let sandbox
+    let cwd
+    let webAppPort
+
+    before(async function () {
+      sandbox = await createSandbox(['mocha', 'jest', '@cucumber/cucumber', 'chai@v4', `selenium-webdriver@${version}`])
+      cwd = sandbox.folder
+
+      webAppPort = await getPort()
+      webAppServer.listen(webAppPort)
+    })
+
+    after(async function () {
+      await sandbox.remove()
+      await new Promise(resolve => webAppServer.close(resolve))
+    })
+
+    beforeEach(async function () {
+      const port = await getPort()
+      receiver = await new FakeCiVisIntake(port).start()
+    })
+
+    afterEach(async () => {
+      childProcess.kill()
+      await receiver.stop()
+    })
+
+    const testFrameworks = [
+      {
+        name: 'mocha',
+        command: 'mocha ./ci-visibility/test/selenium-test.js --timeout 10000'
+      },
+      {
+        name: 'jest',
+        command: 'node ./node_modules/jest/bin/jest --config config-jest.js'
+      },
+      {
+        name: 'cucumber',
+        command: './node_modules/.bin/cucumber-js ci-visibility/features-selenium/*.feature'
+      }
+    ]
```

Comment:
ℹ️ playwright and cypress have their own test runners for browser tests. It wouldn't make a lot of sense to use selenium within playwright or cypress